### PR TITLE
fix gamma handling for GigE cameras

### PR DESCRIPTION
**Pull request by beetleskin**
When connecting to a GigE camera (e.g. aca1920-50gc), the node crashes with "Error while accessing Gamma in PylonCameraImpl<CameraTraitT>": For some camera types, the Gamma settings might not be available to the interface on startup, as they are handled automatically by the camera itself (Auto Gamma). This was already partly handled for some gamma-related function calls, but not for all of them. This fix adds the missing checks. The behaviour is slightly changed: If the gamma is not set via user, gamma remains in auto mode (i.e. controlled by the camera). Once the user tries to set a gamma value, the gamma mode switches to user.

### CODE REVIEW

File: include/pylon_camera/internal/impl/pylon_camera_gige.hpp [hunk 1]
```diff
@@ -115,9 +115,15 @@ bool PylonGigECamera::applyCamSpecificStartupSettings(const PylonCameraParameter
                 << cam_->GainRaw.GetMax()
                 << "] measured in decive specific units.");
 
-        ROS_INFO_STREAM("Cam has gammma range: ["
+        // Check if Gamma is available, print range
+        if( !GenApi::IsAvailable(cam_->Gamma)) {
```
Comment (mgrrx) [inline include/pylon_camera/internal/impl/pylon_camera_gige.hpp]: Could you please re-indent these lines to follow the style guide

**Review by mgrrx — CHANGES_REQUESTED** [review]

**Comment by beetleskin** [comment]
@mgrrx fixed style and squashed it to the commit.

**Comment by marc-up** [comment]
@beetleskin I reviewed your PR, made some adaptions and merged it. Could you please try this out with your GigE color camera and provide me feedback if this solution worked for your case?

**Comment by beetleskin** [comment]
@marceldebout Works like a charm, thanks a lot for the quick merge!

**Pull request merged**